# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../botan/no-tests.patch
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- Botan-2.0.1/src/build-data/makefile/gmake.in.vanilla	2017-03-12 21:24:13.462991374 +0000
+++ Botan-2.0.1/src/build-data/makefile/gmake.in	2017-03-12 21:25:07.658992304 +0000
@@ -21,7 +21,7 @@
 TESTOBJS      = %{test_objs}
 
 # First make target. Will be used by default
-all: $(CLI) $(TEST)
+all: $(CLI)
 
 # Build Commands
 %{lib_build_cmds}
